### PR TITLE
docs for audio routing in android

### PR DESCRIPTION
**Pull request by jashan1498**

### CODE REVIEW

File: docs/android/v2/features/Audio-Output-Routing.mdx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+## Switch audio focus to another device
+
+
+```kotlin
+fun switchAudioOutput(audioDevice: AudioDevice)
+```
+
+invoke this method with appropriate option , to switch the output to that device
+
+`audioDevice` is the [AudioDevice] type through which audio should be routed.
```
Comment (PratimMallick) [inline docs/android/v2/features/Audio-Output-Routing.mdx:17]: Can we mention the types of Audio Device present?

File: docs/android/v2/features/Audio-Output-Routing.mdx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+</Tab>
+
+## Adding a listener
+
+```kotlin
+fun setAudioDeviceChangeListener(audioManageDeviceChangeListener: AudioManagerEvents)
+```
+add a listener to get updates whenever a device is removed or attached to the device.
+`audioManageDeviceChangeListener` -  listner to be passed to be able to listen to device updates.
```
Comment (PratimMallick) [inline docs/android/v2/features/Audio-Output-Routing.mdx:41]: Typo in listener

File: docs/android/v2/features/Audio-Output-Routing.mdx [hunk 3]
```diff
@@ -0,0 +1,82 @@
+---
+title: Audio Output Routing
+nav: 13
+---
+
+Audio Routing can be helpful in the case when user wants to switch output to device other than the default focussed one by OS.
```
[the remark this answers is not in the document]
Comment (jashan1498) [reply docs/android/v2/features/Audio-Output-Routing.mdx]: updated

File: docs/android/v2/features/Audio-Output-Routing.mdx (hunk 2 above)
Comment (jashan1498) [reply docs/android/v2/features/Audio-Output-Routing.mdx]: updated

**Review by PratimMallick — APPROVED** [review]

**Pull request merged**